lib/extractor/docker-archive/index.ts

```diff
@@ -41,7 +41,9 @@ export function getPlatformFromConfig(
 
 export function getDetectedLayersInfoFromConfig(imageConfig) {
   const runInstructions = getUserInstructionLayersFromConfig(imageConfig)
-    .filter((instruction) => !instruction.empty_layer)
+    .filter(
+      (instruction) => !instruction.empty_layer || !instruction.created_by,
```

Comment:
Hey, just check if I got it. We're filtering for instructions that doesn't have `empty_layer` or `created_by` and we're mapping to convert the content from `created_by` from `"# buildkit"` to `""`. Not sure if I'm missing something, but I think it won't work.

Reply:
You're right! Fixed, we're just re running to check 👍 Thanks!